indico/modules/events/timetable/views.py

```diff
@@ -29,3 +39,56 @@ def getJSFiles(self):
 
     def getCSSFiles(self):
         return WPConferenceModifBase.getCSSFiles(self) + self._asset_env['timetable_sass'].urls()
+
+
+@template_hook('meeting-body')
+def _inject_meeting_body(event, **kwargs):
+    event_tz_name = DisplayTZ(session.user, event.as_legacy).getDisplayTZ()
+    event_tz = timezone(event_tz_name)
+    show_date = request.args.get('showDate', 'all')
+    show_session = request.args.get('showSession', 'all')
+    detail_level = request.args.get('detailLevel', 'contribution')
+    view = request.args.get('view')
+
+    children_strategy = joinedload('children')
+    contrib_strategy = children_strategy.joinedload('contribution')
+    subcontrib_strategy = contrib_strategy.joinedload('subcontributions')
```

Comment:
You can simply do this:

```
children_strategy = joinedload('children')
children_strategy.joinedload('contribution').joinedload('subcontributions')
```

All you need to pass to `options` is the `children_strategy`.